titus-supplementary-component/tasks-publisher/src/main/java/com/netflix/titus/supplementary/taskspublisher/TitusClientImpl.java

```diff
@@ -42,19 +49,27 @@
 public class TitusClientImpl implements TitusClient {
     private static final Logger logger = LoggerFactory.getLogger(TitusClientImpl.class);
     private static final String CLIENT_ID = "tasksPublisher";
-    private final JobManagementServiceGrpc.JobManagementServiceStub jobManagementService;
+    public static final int MAX_CACHE_SIZE = 40000;
+    private final JobManagementServiceStub jobManagementService;
+    private JobManagementServiceFutureStub jobManagementServiceFutureStub;
```

Comment:
any reason all these attributes can't be `final`? Not a big deal, but it would make it easier to maintain this code in the future.

Reply:
Done